_posts/2020-08-12-DDoS-criminal-hackers.md

```diff
@@ -0,0 +1,68 @@
+image: images/blogimages/ddos.png
+author: lucyparsonslabs
+teaser: "Documents obtained by LPL appear to confirm DHS is investigating journalists for their reporting"
+tags:
+  - First Amendment
+---
+
+Documents obtained by Lucy Parsons Labs under the Illinois Freedom of Information Act (FOIA)
+appear to confirm the US government is aggressively investigating a journalism
+non-profit as "criminal hackers" for publishing original reporting.
+
+The documents, first obtained and published by the transparency non-profit
+Distributed Denial of Secrets (DDoS), provide an intimate look into police
+agencies and fusion centers across the country. Established after 9/11 to
+disrupt terrorism plots, fusion centers have instead been challenged for massive
+privacy and civil liberties violations. DDoS obtained over 270 gigabytes of data from
+fusion centers and police departments and published the release in what it
+billed the #BlueLeaks release.
+
+Journalists and researchers immediately seized on the Blue Leaks release to
+reveal that police were obsessed with left-wing movements and ignoring
+far-right [domestic extremism](https://theintercept.com/2020/07/15/george-floyd-protests-police-far-right-antifa/).
+Blue Leaks further revealed that law enforcement inflated threats after the
+police murder of George Floyd in Minneapolis with some police stating, without evidence, that "Antifa"
+would carry out [car bombings](https://theintercept.com/2020/06/26/blueleaks-minneapolis-police-protest-fears/).
+The shoddy intelligence of fusion centers has been [well documented](https://www.hsgac.senate.gov/imo/media/doc/10-3-2012%20PSI%20STAFF%20REPORT%20re%20FUSION%20CENTERS.2.pdf)
+for over fifteen years.  
+
+Nonetheless, authorities moved swiftly to crack down on DDoS's right to publish, first by [seizing its server](https://www.infosecurity-magazine.com/news/german-police-seize-blueleaks/)
+in the middle of the night presumably on behalf of the FBI. Homeland Security
+Investigations (HSI), one of the largest agencies within DHS, visited
+unaffiliated individuals hosting an archives of the Blue Leaks and even [offered them money](https://www.vice.com/en_us/article/xg84ww/ice-questions-admin-of-the-eye-archive-site-hosted-blue-leaks) to become informants. Just this week, documents obtained by Lucy Parsons Labs [confirm](https://cdn.muckrock.com/foia_files/2020/08/10/CCSO_00000003.pdf) DHS considers DDoS "criminal hackers" and not journalists.
+
+
+The government's aggressive push to name DDoS as "criminal hackers" is a
+dangerously unprecedented territory that criminalized everyone's right to
+publish online under the First Amendment. It is not illegal to possess and
+publish information obtained by a third party, even if it was obtained
+illegally, under the First Amendment as long as an individual does not
+participate in the illegal activity. This extends even to the publishing of
+classified information, as the Supreme Court
```

Comment:
I would reword slightly. Maybe "As long as the publisher themselves did not participate in illegal activity, possessing and publishing information obtained by a third-party, even obtained illegally, is completely protected under the First Amendment."? 

Also, maybe a reference to the SCOTUS case Bartnicki v. Vopper

Reply:
The verb we care about is the act of publishing of the information even if the information was illegally obtained. I can re-write it a bit and add the reference. I think moving the "publisher" to the front muddles the sentence.

Reply:
Publishing information, even information
[obtained illegally](https://en.wikipedia.org/wiki/Bartnicki_v._Vopper, is
protected under the  First Amendment so long as the publisher does not
participate in the illegal activity.